src/controllers/trip.js

```diff
@@ -0,0 +1,64 @@
+import { Trip as _trip, User } from "../database/models";
+
+const toInclude = {
+  model: User,
+  attributes: ["first_name", "last_name", "email"],
+};
+
+export default class Trip {
+  static async getOne(req, res) {
+    const trip = await _trip.findByPk(req.params.tripId, {
+      include: [
+        { ...toInclude, as: "requester" },
+        { ...toInclude, as: "manager" },
+      ],
+    });
+    if (!trip) {
+      return res.status(404).json({ status: 404, message: "Trip not found" });
+    }
+    if (
+      trip.requester_id !== req.user.id
+      // || trip.manager_id !== req.user.manager_id
+    ) {
+      return res.status(403).json({ status: 403, message: "Trip not yours!" });
+    }
+    return res.status(200).json({ status: 200, data: trip });
+  }
+
+  static async getAll(req, res) {
+    // if (req.user.role !== "manager") {
```

Comment:
Remove commented lines

Reply:
@niyodusengaclement Those were for after User roles were merged because finding all shall be for manager only I think. SO in the future after teammates rebase the don't have to redo my task.